i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/connectors/README.mdx

```diff
@@ -0,0 +1,117 @@
+import ConnectorPlatformIntro from './fragments/_connector_platform_intro.mdx';
+import ConnectorStorageIntro from './fragments/_connector_storage_intro.mdx';
+import ConnectorTypeIntro from './fragments/_connector_type_intro.mdx';
+
+# 🔌 连接器
+
+## 介绍
+
+### 什么是连接器？
+
+_连接器_ 在 Logto 中扮演着重要的角色。在他们的帮助下，Logto 使得终端用户能够使用无密码注册登录和社交账号登录。在网站和应用逐渐渗透到我们的生活中的大背景下，无密码登录和社交登录让用户们能够避免为了管理各种各样的账号而感到头大。
+
+:hushed: 是不是听起来非常引人入胜？ 如果你想要更改一个已有连接器的设置，请参考 [配置连接器](../../recipes/configure-connectors/) 。如果找不到想要更改设置的连接器，你可以根据 [创建连接器](../../recipes/create-your-connector/) 来开发属于你的连接器。
+
+## 组成
+
+连接器数据中有很多属性。
+
+为了提升数据的加载和更新效率，我们把一部分经常更新的连接器数据托管到数据库，剩下的一部分保存在本地缓存中。
+
+<ConnectorStorageIntro />
+
+## 连接器的 _本地储存_：_ConnectorMetadata_
+
+### id
+
+_id_ 是一个用来在 Logto 中 **唯一** 标识连接器的字符串类型参数。
+
+他由连接器的开发者指定，并会被同步到数据库中。
+
+### target
+
+_target_ 是用于区别社交身份源的小写英文字符串。
+
+对于 _短信连接器_ 和 _邮件连接器_ 来说，_target_ 就好像一个人类可读的 ID。
+
+下面看一些关于 _target_ 的样例：_微信连接器网页版_ 和 _微信连接器原生版_ 的 _target_ 是 `wechat`，*阿里云邮件推送连接器*的 _target_ 是 `aliyun-dm`，_阿里云短信连接器_ 的 _target_ 是 `aliyun-sms`。
+
+### type
+
+_type_ 的值应该是下列之一：
+
+<ConnectorTypeIntro />
+
+### platform
+
+_platform_ 的值只能是 `null` 或者下列的字符之一：
+
+<ConnectorPlatformIntro />
```

Comment:
should we give a brief definition for these two? just how/where it's gonna be used.